DataProcessing/DataConverter.cs

```diff
@@ -282,8 +298,12 @@ public void TryAdd(DateTime date, string borrowableShares)
             /// <returns>List of CSV lines, sorted in ascending order by date</returns>
             public IEnumerable<string> ToCsv()
             {
-                return _entries.OrderBy(kvp => kvp.Key)
-                    .Select(kvp => string.Join(",", kvp.Key.ToString("yyyyMMdd"), kvp.Value));
+                return _entries.OrderBy(entry => entry.Item1)
```

Comment:
I think now that we have more columns, in the output file we should add a header with the columns names too, for both date and symbol files 👍. Lean will already handle (skip) comment lines starting with `#`  see https://github.com/QuantConnect/Lean/blob/master/Common/Data/Shortable/LocalDiskShortableProvider.cs#L101